src/pages/AdvancedSearch.vue

```diff
@@ -187,11 +187,20 @@ export default class AdvancedSearchPage extends Vue {
       });
     }
   }
-  private onFormChange(input: { name: string; value: string }) {
+  private async onFormChange(input: { name: string; value: string }) {
     const { name, value } = input;
     let processedName = name;
     let processedVal = inputProcessor(name, value);
 
+    // Fallback
+    if (!processedVal && ["vote", "generatorPublicKey"].includes(name)) {
+      try {
+        processedVal = (await WalletService.find(value)).publicKey;
+      } catch (error) {
+        processedVal = value;
```

Comment:
This should be `processedVal = undefined;` or you end up using the value that is not a public key to perform the search.

Reply:
This was done intentionally and on request by the maintainer to let the search query show 'no results' if no public key is found. If the value would be `undefined`, then the search query would simply return all results.